cmd/minishift/cmd/start.go

```diff
@@ -409,26 +409,75 @@ func exportContainerImages(hostVm *host.Host, version string) {
 	fmt.Println(fmt.Sprintf("-- Exporting of OpenShift images is occuring in background process with pid %d.", exportCmd.Process.Pid))
 }
 
-// calculateDiskSizeInMB converts a human specified disk size like "1000MB" or "1GB" and converts it into Megabits
-func calculateDiskSizeInMB(humanReadableDiskSize string) int {
-	diskSize, err := units.FromHumanSize(humanReadableDiskSize)
+type sizeValidatorFunc func(name string, diskSize string) error
+type sizeConverterFunc func(input int64) int
+
+func calculateHumanSize(sizeValidator sizeValidatorFunc, name string, humanReadableSize string, unit string, sizeConverter sizeConverterFunc) (int, error) {
+	if stringUtils.HasOnlyNumbers(humanReadableSize) {
```

Comment:
Should this not really throw an error? The help for example says 'Use the format <size><unit>'. Also if I right now do something like:

    $ minishift start --memory 5

I get 

    -- Minishift VM will be configured with ...
    Memory:    0 B

The reasons is that MB is assumed as default unit, even tough the default memory is 2GB. That seems inconsistent.